app/views/custom_styles/_inline_css.erb

```diff
@@ -27,7 +27,7 @@ See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
 
-<style>
+<% nonced_javascript_tag do %>
```

Comment:
Shouldn't this be a `hashed_style_tag` instead? [This seems to suggest](https://github.com/twitter/secureheaders/blob/master/docs/hashes.md) that it should be.

Reply:
Not sure what of the article you linked suggests this.

The content is dynamic, thus a statically produced hash (upon asset compilation) would never match. The nonce is generated per-request instead.